### PR TITLE
update template to v2 manifest

### PR DESCRIPTION
**Pull request by karthik2804**

### CODE REVIEW

File: spin.toml [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
+
+[application]
 name = "spin-sqlite-web-cli"
-trigger = { type = "http", base = "/" }
 version = "0.1.0"
+description = "A spin app to explore the the sqlite DB"
+authors = ["karthik2804 <karthik.ganeshram@fermyon.com>"]
+
+[application.trigger.http]
+base = "/"
```
Comment (itowlson) [inline spin.toml:10]: This is deprecated in Spin 2.7.

File: spin.toml [hunk 2]
```diff
@@ -1,36 +1,40 @@
...
+[[trigger.http]]
+id = "trigger-dev"
+component = "dev"
+route = "/static/..."
+
+[component.sql-cli]
 source = "target/spin-sqlite-web-cli.wasm"
+variables = { sqlite_username = "{{ sqlite_username }}", sqlite_password = "{{ sqlite_password }}" }
 exclude_files = ["**/node_modules"]
+allowed_outbound_hosts = ["redis://*:*", "mysql://*:*", "postgres://*:*"]
```
Comment (itowlson) [inline spin.toml]: Shouldn't be needed I would think?

File: spin.toml [hunk 3]
```diff
@@ -1,36 +1,40 @@
...
-[[component]]
+watch = ["assets/sqlite-cli.html", "assets/index.js", "assets/index.css", "src/**/*"]
+
+[component.dev]
 source = { url = "https://github.com/fermyon/spin-fileserver/releases/download/v0.0.3/spin_static_fs.wasm", digest = "sha256:38bf971900228222f7f6b2ccee5051f399adca58d71692cdfdea98997965fd0d" }
-id = "dev"
 files = [{ source = "assets", destination = "/" }]
-[component.trigger]
-route = "/static/..."
+allowed_outbound_hosts = ["redis://*:*", "mysql://*:*", "postgres://*:*"]
```
Comment (itowlson) [inline spin.toml]: Definitely no reason for the fileserver to be accessing any of these!

File: templates/spin-sqlite-explorer/content/spin.toml [hunk 4]
```diff
@@ -1,20 +1,25 @@
...
-[[component]]
+[application.trigger.http]
+base = "/"
+
+[[trigger.http]]
+component = "{{project-name | kebab_case}}"
+route = "/internal/sqlite/..."
+
+[component.{{project-name | kebab_case}}]
 id = "sql-cli"
```
Comment (itowlson) [inline templates/spin-sqlite-explorer/content/spin.toml]: This is no longer allowed in v2 - the ID is given by the table name.

**Review by itowlson — COMMENTED** [review]
At first I thought there was more escaping needed but I now think I was mixing up the template `spin.toml` and the application `spin.toml`!  So just a few nits, which I think will come out during testing anyway!

File: spin.toml (hunk 3 above)
Comment (karthik2804) [reply spin.toml]: Ah these come from using `spin doctor` to migrate to `v2`. Will remove.

**Comment by karthik2804** [comment]
@itowlson I have addressed the review comments and tested to make sure things work. I will merge it.

**Pull request merged**